backend/utils/file_storage/helpers/common_file_helper.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any
+
+from unstract.sdk.file_storage import FileStorageProvider
+from unstract.sdk.file_storage.fs_impl import FileStorage
+from unstract.sdk.file_storage.fs_permanent import PermanentFileStorage
+from unstract.sdk.file_storage.fs_shared_temporary import SharedTemporaryFileStorage
+from utils.file_storage.common_utils import FileStorageUtil
+from utils.file_storage.constants import FileStorageKeys, FileStorageType
+
+
+class FileStorageHelper:
+    # TODO : Optimize this to a singleton class
+    @staticmethod
+    def initialize_file_storage(type: FileStorageType) -> FileStorage:
+        provider_data = FileStorageHelper.load_file_storage_envs()
+        provider = provider_data[FileStorageKeys.FILE_STORAGE_PROVIDER]
+        credentials = provider_data[FileStorageKeys.FILE_STORAGE_CREDENTIALS]
+        if type.value == FileStorageType.PERMANENT.value:
+            file_storage = PermanentFileStorage(
+                provider=provider, credentials=credentials
+            )
+        if type.value == FileStorageType.TEMPORARY.value:
+            file_storage = SharedTemporaryFileStorage(
+                provider=provider, credentials=credentials
+            )
+        file_storage = FileStorage(
```

Comment:
@harini-venkataraman should this be within an `else` block? I see that this ignores whatever was set above

Reply:
@harini-venkataraman this needs to be in else block as said by @chandrasekharan-zipstack .
Also 

 file_storage = FileStorage(
            provider=FileStorageProvider.LOCAL, credentials=credentials
        )